packages/e2e-tests/specs/editor/various/toolbar-roving-tabindex.test.js

```diff
@@ -35,7 +35,7 @@ async function testBlockToolbarKeyboardNavigation(
 	await expectLabelToHaveFocus( 'Move up' );
 	await page.keyboard.press( 'Tab' );
 	await expectLabelToHaveFocus( currentBlockLabel );
-	await pressKeyWithModifier( 'shift', 'Tab' );
+	await focusBlockToolbar();
```

Comment:
This just executes Alt+F10 for us.

Reply:
Shift+Tab now works again, right? So we can revert these lines?